AP-protocol.md

```diff
@@ -307,30 +307,42 @@ TODO
 
 # Simulation Controller
 
-Like nodes, the **Simulation Controller** runs on a thread. It must retain a means of communication with all nodes of the network, even when drones go down.
+Like nodes, the **Simulation Controller** (SC) runs on a thread. It must retain a means of communication with all nodes of the network, even when drones go down. 
+The Simulation controller can send and receive different commands to/from the nodes (drones, clients and servers) through reserved channels. The list of available **commands** is as follows:
 
-### Simulation commands
-
-The Simulation Controller can send the following commands to drones:
-
-`Crash`: This commands makes a drone crash. Upon receiving this command, the drone’s thread should return as soon as possible.
+```rust
+pub enum Command {
+    AddChannel(NodeId, Sender<Packet>),
+    RemoveChannel(NodeId),
+    Crash,
+    ChangePdr(f32),
+    SentMessage(Packet),
+    DroppedMessage(Packet)
```

Comment:
I would names into 'SentPacket' and 'DroppedPacket' or fragment, since the message should be the whole thing, and the drones only transports the fragments.
I'd also add the id of the drone the packet is sent to inside 'SentMessage', so that the sim controller can easily display the sending and receiving drone with the message.
And the pdr in 'DroppedMessage' might be useful to display too.

Reply:
I agree with you, Ill change them and add the PDR in DroppedMessage

Reply:
I just noticed that within the SourceRoutingHeader (in Packet) we have the hop_index, thus we can retrieve the Node involved in SentPacket and DroppedPacket from there. 

For now, ill just add the PDR to DroppedPacket and refactor their names. Then let me know what you think.